features/facebookImport/src/model/caching.js

```diff
@@ -0,0 +1,80 @@
+import EntityDeserializer from "./entities/utils/entity-serialisation";
+
+const namespace = "http://polypoly.coop/schema/fbImport/";
+
+export default class ExportDataCache {
```

Comment:
I see several possible problems with this
1. There's no way to limit the size of the cache at the logic level. That means that it's not really a cache, but rather a fast (ish) storage.
2. The ish is also an issue. Depending on how the triple store is implemented, we might end up with permanent storage accessed through the browser, so no real guarantee that it's going to be faster than accessing the file.
3. Implementation is always an issue; depending on how it's implemented, size is going to be limited. So again point 1: not a real cache.